cmd/root.go

```diff
@@ -104,18 +103,23 @@ func getRemoteVersion(req string) (string, error) {
 		return "", err
 	}
 
+	if len(body) > 50 {
```

Comment:
Should we do something like pattern based here instead of length?

Reply:
After the check we are doing unmarshall of the xml contents so pattern matching will fail if the expected tags are not present. Hence did not do that explicitly.